plugins/dell-k2/fu-dell-k2-ec.c

```diff
@@ -826,8 +828,8 @@ fu_dell_k2_ec_write_firmware(FuDevice *device,
 		return FALSE;
 	fu_progress_step_done(progress);
 
-	/* appease daemon as a device reset is required */
-	fu_device_set_version(device, fu_firmware_get_version(firmware)); /* nocheck:set-version */
+	/* check version is not required */
+	fu_device_add_flag(device, FWUPD_DEVICE_FLAG_INSTALL_SKIP_VERSION_CHECK);
```

Comment:
but this is only when UOD flow is used, right?  So shouldn't it be conditional?

Reply:
It will go for UOD at all times, no online-reset i.e. no activate command unless it become possible. 

We need device flag `uod-off` to disable it as it isnt fully ready yet.